charts/cardano-db-sync/values.yaml

```diff
@@ -3,26 +3,31 @@
 
 cardano_network: mainnet
 
-cardano_node: {}
-  # host: node-mainet.svc.cluster.local
-  # port: 3001
+cardano_node:
+  host: node-mainet.svc.cluster.local
```

Comment:
This isn't going to be a valid hostname in any cluster, so we probably shouldn't make it the default